client_test.go

```diff
@@ -0,0 +1,169 @@
+package xsolla
+
+import (
+	"fmt"
+	"github.com/joho/godotenv"
+	"log"
+	"os"
+	"strconv"
+	"testing"
+)
+
+
+func newTestClient() *Client {
+	err := godotenv.Load(".env")
+	if err != nil {
+		log.Fatalf("Error loading .env file")
```

Comment:
Avoid doing this as not everyone will use a .env file. It may be setup via an export in the line or bash profile for example.

Reply:
The client should instead return an error stating it's not configured.